appcues/src/main/java/com/appcues/ui/AppcuesOverlayViewManager.kt

```diff
@@ -60,8 +60,12 @@ internal class AppcuesOverlayViewManager(
         AppcuesActivityMonitor.activity?.removeOverlayView()
     }
 
+    private fun setOverlayVisible(isVisible: Boolean) {
+        AppcuesActivityMonitor.activity?.updateOverlayVisibility(isVisible)
+    }
+
     private fun Activity.addOverlayView(): Boolean {
-        viewModel = AppcuesViewModel(scope, renderContext, ::onCompositionDismiss)
+        viewModel = AppcuesViewModel(scope, renderContext, ::onCompositionDismiss, ::setOverlayVisible)
```

Comment:
this part, and related helper function, may need to evolve to compile correctly in the new embed framework - though embeds will not necessarily have any sort of retry needed, not actually use the hide/show.